pkg/paths/path.go

```diff
@@ -24,13 +26,15 @@ type AllPathParams struct {
 
 // GetAllParams fetches the path patams from c and returns them
 func GetAllParams(c buffalo.Context) (*AllPathParams, error) {
+	const op errors.Op = "paths.GetAllParams"
 	mod, err := GetModule(c)
 	if err != nil {
-		return nil, err
+		return nil, errors.E(op, err)
 	}
 	version := c.Param("version")
 	if version == "" {
-		return nil, fmt.Errorf("version not found")
+		return nil, errors.E(op, "version not found")
```

Comment:
Same here, present from another commit but the string passed could better describe that the version parameter is missing and not the version itself.